### PR TITLE
Feature/styling optimization

### PR DESCRIPTION
**Pull request by yiliansource**
Lots of CSS changes, as well as view and page optimization.

### CODE REVIEW

File: Core/Database/Helpers/EventHelper.cs [hunk 1]
```diff
@@ -31,13 +31,24 @@ public Event CreateEvent(Event e, UserAccount host)
             return e;
         }
 
+        /// <summary>
+        /// Deletes the specified event.
+        /// </summary>
+        public bool DeleteEvent(Event e)
+        {
+            context.Events.Remove(e);
```
Comment (AquaXV) [inline Core/Database/Helpers/EventHelper.cs:39]: What happens to the entries if you delete an event? Do they become stale or are they deleted automatically as well?

File: Pages/Shared/_EventPageHeader.cshtml [hunk 2]
```diff
@@ -1,73 +1,70 @@
...
+        <p class="event-description">@Html.Raw(Model.Event.Description.Replace(Environment.NewLine, "<br />"))</p>
+    }
+
     <div class="event-datebox">
-        @switch(eventState)
+        @switch(Model.EventState)
         {
             case EventState.Hidden:
-                <div class="timer event-timer" --unix-timestamp="@Model.RevealDate.ToUnixSeconds()">Reveal in</div>
+                <div class="timer event-timer" --unix-timestamp="@Model.Event.RevealDate.ToUnixSeconds()">Reveal in</div>
```
Comment (AquaXV) [inline Pages/Shared/_EventPageHeader.cshtml:23]: If an event is hidden it should not even be showing "Reveal in". It is hidden for a reason.

File: Core/Database/Helpers/EventHelper.cs (hunk 1 above)
Comment (yiliansource) [reply Core/Database/Helpers/EventHelper.cs:39]: In my testing scenarios, the entries were automatically deleted.
We might need to test this in a production scenario.

File: Pages/Shared/_EventPageHeader.cshtml (hunk 2 above)
Comment (yiliansource) [reply Pages/Shared/_EventPageHeader.cshtml:23]: That is only shown to event moderators.

**Review by AquaXV — APPROVED** [review]

**Pull request merged**